cms/djangoapps/contentstore/rest_api/v2/views/home.py

```diff
@@ -81,11 +83,11 @@ def get(self, request: Request):
         }
         ```
         """
-
         courses, in_process_course_actions = get_course_context_v2(request)
-        courses_context = {
-            "courses": courses,
-            "in_process_course_actions": in_process_course_actions,
-        }
-        serializer = CourseHomeTabSerializerV2(courses_context)
-        return Response(serializer.data)
+        paginator = PageNumberPagination()
+        courses_page = paginator.paginate_queryset(courses, self.request, view=self)
+        serializer = CourseHomeTabSerializerV2({
+            'courses': courses_page,
+            'in_process_course_actions': in_process_course_actions,
+        })
+        return paginator.get_paginated_response(serializer.data)
```

Comment:
How can we configure the page size?

Reply:
We're using the DRF default for simplicity: https://github.com/openedx/edx-platform/blob/43747ee8f08046a8892b5ad6cc9534fc0c2a896d/lms/envs/common.py#L3364-L3391

Reply:
@mariajgrimaldi I would be great to have the page size, I'm not sure what is it the default page size value